tests/puppeteer/pages/commonPage.js

```diff
@@ -3,14 +3,31 @@ module.exports = class CommonPage {
     this.page = page;
   }
 
+  /**
+   * Get page title
+   * @returns {Promise<*>}
+   */
   async getPageTitle() {
     return this.page.title();
   }
 
+  /**
+   * Go to URL
+   * @param URL
+   * @returns {Promise<void>}
+   */
   async goTo(URL) {
```

Comment:
```suggestion
  async goTo(url) {
```
Could you use lowercase here please?